pkg/sql/opt/memo/testdata/stats/join

```diff
@@ -1425,7 +1425,7 @@ full-join (cross)
 expr
 (SemiJoin
     (Values
-      [ (Tuple [ (Const 1) (Const 2) ] "tuple{int}" ) ]
+      [ (Tuple [ (Const 1 "int") (Const 2 "int") ] "tuple{int}" ) ]
```

Comment:
Could these be MakeIntConst?

Reply:
No, there are no custom functions in exprgen